app/helpers/kaui/date_helper.rb

```diff
@@ -13,7 +13,7 @@ def format_date(date, timezone="Pacific Time (US & Canada)")
 
       # If not, convert into account timezone and return the date part only
       parsed_date = DateTime.parse(date.to_s).in_time_zone(timezone)
-      parsed_date.to_s(:date_only)
+      parsed_date.to_s(:pretty)
```

Comment:
There are two aspects in this change:

* Display of `DateTime` vs `LocalDate`. We should discuss the exact requirements in more details (do you want to start an email thread? I have no context on this) and decide whether (and where, e.g. payment screens only) the change makes sense. Note that if we decide to switch *some* of these to `DateTime`, we also need to think about the timezone (almost all of the screens display the date and time in the account timezone, not UTC) to be consistent.
* Format of the date and time: if you want something pretty and/or customized, this should be done in a locale configuration file (I don't think you need to fork it for this).